sys/kern/kasan.c

```diff
@@ -299,7 +308,6 @@ void __asan_unregister_globals(struct __asan_global *globals, size_t n) {
   /* never called */
 }
 
-/* Note: alloca is currently used in strntoul and test_sleepq_sync functions */
```

Comment:
Is that no longer true?

Reply:
Yes, this message is obsolete.

Reply:
Just FYI, this wasn’t about us explicitly using alloca in the implementation but about the compiler using alloca while compiling those functions